def data_error():
    return 0
